crates/nu-command/src/filters/par_each.rs

```diff
@@ -49,39 +54,34 @@ impl Command for ParEach {
     fn examples(&self) -> Vec<Example> {
         vec![
             Example {
-                example: "[1 2 3] | par-each {|| 2 * $in }",
+                example: "[1 2 3] | par-each {|e| $e * 2 }",
                 description:
                     "Multiplies each number. Note that the list will become arbitrarily disordered.",
                 result: None,
             },
             Example {
-                example: r#"[foo bar baz] | par-each {|e| $e + '!' } | sort"#,
-                description: "Output can still be sorted afterward",
-                result: Some(Value::list(
-                    vec![
-                        Value::test_string("bar!"),
-                        Value::test_string("baz!"),
-                        Value::test_string("foo!"),
-                    ],
-                    Span::test_data(),
-                )),
+                example: r#"[1 2 3] | par-each -k {|e| $e * 2 }"#,
+                description: "Multiplies each number, keeping an original order",
+                result: Some(Value::test_list(vec![
+                    Value::test_int(2),
+                    Value::test_int(4),
+                    Value::test_int(6),
+                ])),
             },
             Example {
-                example: r#"1..3 | enumerate | par-each {|p| update item ($p.item * 2)} | sort-by item | get item"#,
-                description: "Enumerate and sort-by can be used to reconstruct the original order",
-                result: Some(Value::list(
-                    vec![Value::test_int(2), Value::test_int(4), Value::test_int(6)],
-                    Span::test_data(),
-                )),
```

Comment:
I think I'd keep this example. It's still valid isn't it?

Reply:
I thought that the new feature replaces this example, so it is no longer necessary.
At least, if we keep this example, it should be fixed to:
```nu
1..3 | enumerate | par-each {|p| update item ($p.item * 2)} | sort-by index | get item
#                                                                     ^^^^^
#                                                                     instead of `item`
```
Because "reconstruct the original order" will not apply on an input like `3..1`

Reply:
If we use `enumerate` there should be an item and index. But your example should be sort-by item vs index, right?

I also agree that your PR is better and easier, but we shouldn't remove the ability to `enumerate` things.

```nushell
 1..3 | enumerate | each {|r| $"($r.index) - ($r.item)"}
╭─┬─────╮
│0│0 - 1│
│1│1 - 2│
│2│2 - 3│
╰─┴─────╯
```

Reply:
I restored the original example